engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -384,6 +384,60 @@ paths:
       security:
         - google_oauth:
             - openid
+  '/workflows/{version}/callcaching/diff':
+    get:
+      summary: Return the hash differential between to calls
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
+        - name: workflowA
+          description: Workflow Id of the first workflow
+          required: true
+          type: string
+          in: query
+        - name: callA
+          description: Call FQN (including workflow name) of the first call
+          required: true
+          type: string
+          in: query
+        - name: indexA
+          description: Index of the first call
+          required: false
```

Comment:
@mcovarr Just to be clear, if it is required it means we would always have to give "-1" for non sharded calls right ? I honestly don't know which one is more natural